linux_os/guide/system/auditing/coreos_audit_option/rule.yml

```diff
@@ -19,7 +19,7 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@ocp4: CCE-83550-4
```

Comment:
Changing ocp4 to rhcos seems good, just making sure the CCE number is correct (I have no idea where they come from..)

Reply:
Nevermind this identifier was removed from the RHEL rule, so it seems correct